composer.json

```diff
@@ -26,7 +26,8 @@
     },
     "require-dev": {
         "mikey179/vfsStream": "~1.5.0",
-        "symfony/var-dumper": "~3.1.3"
+        "symfony/var-dumper": "~3.1.3",
+        "phpunit/phpunit": "^5.4"
```

Comment:
we do follow semver, we just don't fall for the dynamics and errors sometimes happen